### PR TITLE
Update DSTU 2 documentation for Immunization.

### PR DESCRIPTION
**Pull request by jfrhong**
- Updated terminology bindings
- Updated example bundle

### CODE REVIEW

File: lib/dstu2_resources.rb [hunk 1]
```diff
@@ -361,104 +361,270 @@ module Resources
...
+              "id": "HM4234847",
+              "meta": {
+                "versionId": "0",
+                "lastUpdated": "2016-05-13T12:03:36-0500"
+              },
+              "text": {
+                "status": "generated",
+                "div": "&lt;div>&lt;p>&lt;b>Immunization&lt;/b>&lt;/p>&lt;p>&lt;b>Vaccine&lt;/b>: Hepatitis B&lt;/p>&lt;p>&lt;b>Date Attempted&lt;/b>: 2016-05-13T12:03:36-0500&lt;/p>&lt;p>&lt;b>Given&lt;/b>: No&lt;/p>&lt;p>&lt;b>Administered by&lt;/b>: Dr. Who&lt;/p>&lt;p>&lt;b>Reasons Not Given&lt;/b>: Patient Refuses&lt;/p>&lt;/div>"
+              },
+              "status": "on-hold",
```
Comment (daliboz) [inline lib/dstu2_resources.rb]: Is this still part of our returned value set? Per conversations around Argonaut, these statuses were removed.

**Comment by daliboz** [comment]
Other than the question around the value set for status, +1.

File: lib/dstu2_resources.rb (hunk 1 above)
Comment (jfrhong) [inline lib/dstu2_resources.rb]: We are not currently honor the Argonaut value set. There is a JIRA logged to switch to it.

**Comment by cerbs** [comment]
+1, Other than the obligatory observation that 'Dr. Who' should be 'The Doctor'

**Pull request merged**